backend/test/api/scores_test.clj

```diff
@@ -0,0 +1,55 @@
+(ns api.scores-test
+  (:require [expectations.clojure.test :refer :all]
+            [api.base :as api]
+            [test-helper :as helper]
+            [configs.hands :as hands]))
+
+(defexpect rows-won
+  (expect
+    #(= [3 1] 
+        (:scores %))
+    (helper/play-as-api
+      (fn [i] (mod i 4))
+      (fn [i] 0)))
+  (expect
+    #(= [2 2]
+        (:scores %))
+    (helper/play-as-api
+      (fn [i] (mod i 2))
+      (fn [i] (+ (mod i 2) 2)))))
+
+(defexpect power-in-rows
+  ; Game begins with scores at 0
+  (expect
+    [[0 0] [0 0] [0 0] [0 0] [0 0]]
+    (let [game-state (api/create-game)
+          game-id (:game-id game-state)
+          player-id (:player-id game-state)
+          opponent-id (:player-id (api/add-player game-id))]
+      (:rows-power (api/get-game game-id player-id))))
+  (expect
+    [[(helper/default-hand-power 0) (helper/default-hand-power 1)]
```

Comment:
both `helper` and `default-hand-power` aren't very descriptive, I think, since I believe this generates a card of power 0 (correct me if I'm wrong)

Reply:
You're wrong.
This returns the power of the nth card in the `default-hand`

Reply:
ok, absolutely definitely change the name to something understandable, please, please, like `hand/power-of-nth` or something

Reply:
`hand/power-of-nth index hand`, so it can be also reused for different hands (we will have them pretty soon, when introducing power-changing abilities).